servicetalk-bom/build.gradle

```diff
@@ -71,8 +71,8 @@ publishing {
     repositories {
       maven {
         name = "sonatype"
-        def releasesRepoUrl = "https://oss.sonatype.org/service/local/staging/deploy/maven2"
-        def snapshotsRepoUrl = "https://oss.sonatype.org/content/repositories/snapshots"
+        def releasesRepoUrl = "https://ossrh-staging-api.central.sonatype.com/service/local/staging/deploy/maven2"
+        def snapshotsRepoUrl = "https://central.sonatype.com/repository/maven-snapshots/"
```

Comment:
Just double checking: should it have slash at the end?

Reply:
It looks like the docs show a slash at the end, and I've made them consistent with that.
https://central.sonatype.org/publish/publish-portal-ossrh-staging-api/#plugins-tested-for-compatibility